README.md

```diff
@@ -178,6 +188,11 @@ be passed using `[]`, for example: `:token-name[pretty]` would pass the string
 Normally formats are defined using `morgan.format(name, format)`, but for certain
 advanced uses, this compile function is directly available.
 
+The function returned takes arguments `tokens` , `req`, and `res` where `tokens`
+refers to `morgan` itself. If a log should be skipped the function will return
+`null`. The function returned or a custom function can be passed directly to 
```

Comment:
I still think it feels like you just stuck this in a random spot (when reading the docs--why do I need to read the `.compile()` function to understand what to pass into `format`? The current docs direct to https://github.com/expressjs/morgan#predefined-formats not the compile() function.